markdowns/workingGroup/gettingInvolved.md

```diff
@@ -1,3 +1,10 @@
-# Getting Involved
+# Participate And Keep In Touch
```

Comment:
Keeping in sync with w3c did website. also topic must be in sync with the file name.

Reply:
OK